.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -2,4 +2,5 @@ blank_issues_enabled: false
 contact_links:
   - name: Questions
     url: https://discord.gg/Byjr6rdBUZ
-    about: Any questions or comments you have, let us know on our Discord server.
\ No newline at end of file
+    about: Any questions or comments you have, let us know on our Discord server.
+    
```

Comment:
Looks like there are _two_ extra lines now. GitHub doesn't display the endline.
```suggestion
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

Reply:
I just added a single extra line .

Reply:
Hmm, well, GitHub shows an extra line there. It looks like it has a trailing whitespace.

Reply:
<img width="1440" alt="Screenshot 2022-06-18 at 9 57 44 AM" src="https://user-images.githubusercontent.com/90490253/174422719-384f8f1c-9b12-48fe-ab98-2ae20a7928a6.png">